test/new-e2e/examples/vm_test.go

```diff
@@ -25,7 +25,7 @@ type vmSuite struct {
 
 // TestVMSuite runs tests for the VM interface to ensure its implementation is correct.
 func TestVMSuite(t *testing.T) {
-	suiteParams := []e2e.SuiteOption{e2e.WithProvisioner(awshost.ProvisionerNoFakeIntake(awshost.WithEC2InstanceOptions(ec2.WithOS(os.WindowsDefault))))}
+	suiteParams := []e2e.SuiteOption{e2e.WithProvisioner(awshost.ProvisionerNoAgentNoFakeIntake(awshost.WithEC2InstanceOptions(ec2.WithOS(os.WindowsDefault))))}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
I'm splitting up this in a follow up PR, this example is for a simple basic vm, the test is currently broken as windows doesn't know what `whoami` is

Reply:
My bad, it was probably me doing weird things and test failed elsewhere ! 

I will still split this test as I would like a basic vm test here, default os, and another example to show the file system helper. It wold help docs